typescript/token/hardhat.config.ts

```diff
@@ -12,6 +12,12 @@ module.exports = {
     compilers: [
       {
         version: '0.8.16',
+        settings: {
+          optimizer: {
+            enabled: true,
+            runs: 200, // Optimizer runs were added as the HypERC20Votable contract size exceeds the max smart contract size
```

Comment:
Actually while you are at this, can you set the version to 0.8.17 and optimizations to 999999 as the core package https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/solidity/hardhat.config.ts#L10

Reply:
Ok will do that